script.js

```diff
@@ -1 +1,9 @@
 "use strict";
```

Comment:
why use strict?

Reply:
The tutorials say that using strict mode is mandatory for beginners to avoid mistakes